spec/tapioca/cli_spec.rb

```diff
@@ -5,1289 +5,52 @@
-        end
-      end
-
-      describe("with excluded files") do
-        it "advises of removed file(s) and returns exit_status 1" do
-          output = execute("sync", "--verify", exclude: "foo bar")
-
-          assert_equal(output, <<~OUTPUT)
-            Checking for out-of-date RBIs...
-
-            RBI files are out-of-date. In your development environment, please run:
-              `bin/tapioca sync`
-            Once it is complete, be sure to commit and push any changes
-
-            Reason:
-              File(s) removed:
-              - #{outdir}/bar@0.3.0.rbi
-              - #{outdir}/foo@0.0.1.rbi
-          OUTPUT
-          assert_includes($?.to_s, "exit 1") # rubocop:disable Style/SpecialGlobalVars
-
-          # Does not actually modify anything
-          assert_path_exists("#{outdir}/foo@0.0.1.rbi")
-          assert_path_exists("#{outdir}/bar@0.3.0.rbi")
-        end
-      end
-
-      describe("with added/removed/changed files") do
-        before do
-          FileUtils.rm("#{outdir}/foo@0.0.1.rbi")
-          FileUtils.touch("#{outdir}/outdated@5.0.0.rbi")
-          FileUtils.mv("#{outdir}/bar@0.3.0.rbi", "#{outdir}/bar@0.2.0.rbi")
-        end
-
-        it "advises of added/removed/changed file(s) and returns exit_status 1" do
-          output = execute("sync", "--verify")
-
-          assert_equal(output, <<~OUTPUT)
-            Checking for out-of-date RBIs...
-
-            RBI files are out-of-date. In your development environment, please run:
-              `bin/tapioca sync`
-            Once it is complete, be sure to commit and push any changes
-
-            Reason:
-              File(s) added:
-              - #{outdir}/foo@0.0.1.rbi
-              File(s) changed:
-              - #{outdir}/bar@0.3.0.rbi
-              File(s) removed:
-              - #{outdir}/outdated@5.0.0.rbi
-          OUTPUT
-          assert_includes($?.to_s, "exit 1") # rubocop:disable Style/SpecialGlobalVars
-
-          # Does not actually modify anything
-          refute_path_exists("#{outdir}/foo@0.0.1.rbi")
-          assert_path_exists("#{outdir}/outdated@5.0.0.rbi")
-          assert_path_exists("#{outdir}/bar@0.2.0.rbi")
-        end
-      end
-    end
-  end
-
-  describe("deprecations") do
-    it "prints the correct deprecation message with -c" do
-      output = execute("dsl", ["-c", "foo"])
-      assert_includes(output, "DEPRECATION: The `-c` and `--cmd` flags will be removed in a future release.")
-    end
-
-    it "prints the correct deprecation message with --cmd" do
-      output = execute("dsl", ["--cmd", "foo"])
-      assert_includes(output, "DEPRECATION: The `-c` and `--cmd` flags will be removed in a future release.")
-    end
-
-    it "doesn't print the correct deprecation message with no flag" do
-      output = execute("dsl")
-      refute_includes(output, "DEPRECATION: The `-c` and `--cmd` flags will be removed in a future release.")
+module Tapioca
+  class CliSpec < Minitest::HooksSpec
```

Comment:
Split Tapioca::CliSpec into a module and a nested class to follow our style guide.